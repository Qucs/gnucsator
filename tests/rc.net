# Qucs 0.0.19

R:R1 _net0 _net1 R="50 Ohm" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
VProbe:Pr1 _net1 gnd
.TR:TR1 Type="lin" Start="0" Stop="10m" Points="11" IntegrationMethod="Trapezoidal" Order="2" InitialStep="1 ns" MinStep="1e-16" MaxIter="150" reltol="0.001" abstol="1 pA" vntol="1 uV" Temp="26.85" LTEreltol="1e-3" LTEabstol="1e-6" LTEfactor="1" Solver="CroutLU" relaxTSR="no" initialDC="yes" MaxStep="0"
Vrect:V1 _net0 gnd U="1 V" TH="1 ms" TL="1 ms" Tr="1 ns" Tf="1 ns" Td="0 ns"
C:C1 _net1 gnd C="1 pF" V=""
