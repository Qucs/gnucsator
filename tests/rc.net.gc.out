<Qucs Dataset 0.0.19>
<indep time 0>
</indep>
